### DISCUSSION TITLE
Applying custom CSS

### DISCUSSION THREAD

**Discussion by jpatterson-7** [Budibase/budibase#2391] [Help] [answered]
Is there a knack or technique to apply custom CSS? I tried applying the following just to see if I can get it to work but it doesn't do anything. It does work when I apply it via the style editor on the page once it is rendered. 

Are there only specific elements that I can interact with? Or any specific level that it needs to be applied at?

```
.spectrum-Table td div{
  color: green !important;
}
```

**Comment by shogunpurple** [comment, marked as answer]
Hi @jpatterson-7 - custom CSS is applied inline so you cannot style child elements unfortunately. You should be able to use the conditional UI feature to apply certain settings/styles based on conditions if that is useful to you.

**Reply by kevmodrome** [reply]
You can also modify the custom css properties that spectrum uses. To find which one to modify you need to inspect the application.

**Reply by jpatterson-7** [reply]
Ah ok, cheers. I was trying to apply width settings to tables as I'm not able to see all of the content even if the table is full width. I was planning on trying to apply the following which would have worked perfectly :( 

`.spectrum-Table td div{
  width: 100%;
}`

Thanks anyway 👍

**Reply by ttelik** [reply]
@jpatterson-7 Did you happen to find another way of adjusting the table width?
